modules.d/01systemd-resolved/module-setup.sh

```diff
@@ -50,6 +50,7 @@ install() {
     # Install the hosts local user configurations if enabled.
     if [[ $hostonly ]]; then
         inst_multiple -H -o \
+            "$systemdutilconfdir"/resolv.conf \
```

Comment:
I don't know anything about how `systemd-resolved` works, but does `/etc/systemd/resolv.conf` really exist? I thought the old standard was `/etc/resolv.conf`, and that is the only one I find on my Fedora vms, symlinked to `/run/systemd/resolve/stub-resolv.conf`.

Reply:
I have no idea. I've added it for compatibility purposes - i.e. make the approach standardized - everything to load from the same paths; as it has `        "$systemdutildir"/resolv.conf \` entry, I've simply translated it into the hostonly variant.

Reply:
I could not find a reference to `"$systemdutildir"/resolv.conf` in [systemd-resolved.service](https://github.com/systemd/systemd/blob/main/man/systemd-resolved.service.xml)

Reply:
I could not find a reference to "$systemdutildir"/resolv.conf in systemd-resolved.service

Reply:
I suppose `/etc/resolv.conf` is used by the process itself - I would simply continue with this non-breaking change if you don't mind.